dynamiq/evaluations/metrics/bleu_score.py

```diff
@@ -0,0 +1,138 @@
+import logging
+from functools import cached_property
+from typing import Callable
+
+from pydantic import BaseModel, ConfigDict, PrivateAttr, computed_field, model_validator
+
+logger = logging.getLogger(__name__)
```

Comment:
Let's use our logger for evaluations as well:
`from dynamiq.utils.logger import logger`